ImmuDB4Net/ImmuClient.cs

```diff
@@ -354,6 +504,12 @@ public async Task<List<string>> Databases()
     // ========== GET ==========
     //
 
+    /// <summary>
+    /// Retrieves the value for a specific key at a transaction ID.
+    /// </summary>
+    /// <param name="key">The lookup key</param>
+    /// <param name="tx">The transaction id</param>
+    /// <returns>An <see cref="Entry"/> object. Most often the Value field is used.</returns>
     public async Task<Entry> GetAtTx(byte[] key, ulong tx)
```

Comment:
Why this is `GetAtTx` where other methods overload `Get` ?

Reply:
Renamed. Most likely 'borrowed' from Java SDK